pygmt/tests/test_clib_virtualfiles.py

```diff
@@ -137,3 +143,37 @@ def test_open_virtual_file():
         bounds = "\t".join([f"<{col.min():.0f}/{col.max():.0f}>" for col in data.T])
         expected = f"<matrix memory>: N = {shape[0]}\t{bounds}\n"
         assert output == expected
+
+
+@pytest.mark.benchmark
```

Comment:
The test was moved to `test_clib_virtualfile_from_vectors` in #3512.

Reply:
Moved in 6ad6eb96d7d0a0bcbe8e01386d80059a6d70a104